kolibri/plugins/learn/assets/src/actions.js

```diff
@@ -36,7 +36,35 @@ const fetchNodes = ({ dispatch }, id) => {
   });
 };
 
+
+const searchNodes = ({ dispatch }, params, page) => {
+  // Get the collection from ContentNodeResource.
+  const contentCollection = Kolibri.resources.ContentNodeResource.getCollection();
+  const wantedFields = ['pk', 'title', 'kind', 'instance_id', 'content_id', 'description', 'files'];
```

Comment:
since it's type as search, I prefer the quickest reponse possible.
btw, description is not needed. I will get rid of it from the wantedFields